extension.js

```diff
@@ -18,9 +18,9 @@ function OnStatusBarUpdate( textEditor ) {
                 filePath = vscode.workspace.asRelativePath(textEditor.document.fileName)
                 filePath = path.normalize(filePath)
             }
-            sb.tooltip = 'Copy active file to clipboard';
-            if (config.revealFile) {
-                sb.tooltip = 'Reveal file';
+            if(config.showFolderAsProjectName) {
+                filePath = path.normalize(filePath)
+                filePath = path.dirname(filePath).split(config.showFolderAsProjectName)[0].split('/').slice(-1)[0]
```

Comment:
I haven't investigated, but I'm pretty sure the VSCode api can be used to get project and/or workspace names and that should be used instead.